account-kit/wallet-client/src/experimental/actions/requestQuoteV0.ts

```diff
@@ -56,7 +49,7 @@ export async function requestQuoteV0<
         ...params,
         chainId: toHex(client.chain.id),
         from,
-        ...(params.returnRawCalls ? {} : { capabilities }),
+        ...(capabilities && { capabilities }),
```

Comment:
Interesting, I haven't seen this syntax form before. I'm guessing it's equivalent to `...(capabilities ? { capabilities } : []),`, right?